src/components/layout/header-scrolling.js

```diff
@@ -39,19 +28,20 @@ const propTypes = {
 class HeaderScrolling extends Component {
     constructor(props) {
         super(props);
-        const {size} = props;
-        const storeState = this._getStateFromStore();
-        storeState.size = size;
+        let storeState = this._getStateFromStore()
```

Comment:
Seems pretty weird to me to declare that variable `const` and mutate it on the following line.
(Even though it's a reference, I know).